mobile/src/libs/components/toast/toast.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Toast, { ErrorToast } from 'react-native-toast-message';
+
+import { toastStyles } from './styles';
+
+const toastConfig = {
+  error: (properties: object): React.ReactNode => (
+    <ErrorToast
+      {...properties}
+      style={toastStyles.errorToast}
+      text1Style={toastStyles.text1}
+      text2Style={toastStyles.text2}
+    />
+  ),
+};
+
+const ToastComponent: React.FC = () => {
```

Comment:
```suggestion
const Toast: React.FC = () => {
```

Reply:
If I change this name it will conflict with the Toast name from the package react-native-toast-message
How can I handle that?

Reply:
Use import LibraryToast and name component as Toast)